HealthMonitoring.SelfHost/Controllers/EndpointsController.cs

```diff
@@ -22,6 +22,14 @@ public EndpointsController(IEndpointRegistry endpointRegistry, IEndpointStatsRep
             _endpointStatsRepository = endpointStatsRepository;
         }
 
+        [Route("api/endpoints/health/statuses")]

```

Comment:
it does not belong here. The list of statuses is static and does not belong to endpoints resource(s) I think.

I am not sure however where it belongs to. /api/config ? /api/monitors ?